packages/cyberchef.vm/tools/chocolateyinstall.ps1

```diff
@@ -19,13 +19,13 @@ try {
   Install-ChocolateyZipPackage @packageArgs
   VM-Assert-Path $toolDir
 
+  $iconLocation = VM-Create-Ico (Join-Path $toolDir "images\cyberchef-128x128.png") # Create .ico for cyberchef icon
```

Comment:
@emtuls have you forgotten to push the updated changes? :thinking: This code is still creating the `.ico` instead of using https://raw.githubusercontent.com/gchq/CyberChef/18159ce806e63286019a5a6828422a992fbb253b/src/web/static/images/favicon.ico that seems to looks better and does not require the extra function